editor/editor_node.cpp

```diff
@@ -3874,7 +3876,7 @@ Error EditorNode::load_scene(const String &p_scene, bool p_ignore_broken_deps, b
 		Ref<SceneState> state = sdata->get_state();
 		state->set_path(lpath);
 		new_scene->set_scene_inherited_state(state);
-		new_scene->set_scene_file_path(String());
+		new_scene->set_scene_file_path(lpath);
```

Comment:
@godotengine/import Does anyone know what this does? I did this to cover a case where opening a nested scene file loses a reference to the blend file as per the test case when updating.